fips_oop_demo.py

```diff
@@ -24,7 +28,7 @@ def main(config_file, json_file, no_download_algs):
     # dset = FIPSDataset({}, Path('./fips_test_dataset'), 'small dataset', 'small dataset for keyword testing')
 
     # Load metadata for certificates from CSV and HTML sources
-    dset.get_certs_from_web(json_file=json_file, redo=True)
+    dset.get_certs_from_web(json_file=json_file, redo=False)
```

Comment:
This `redo` argument changes with each PR I feel like. Maybe make it a command-line argument?